Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -595,6 +623,208 @@ def alert_to_incident_context(alert: Dict[str, Any]) -> Dict[str, Any]:
+            entry = reopen_incident_in_xsoar(remote_alert_id)
+            return entry
+        return None
+    return None
+
+
+def close_alert_in_prisma_cloud(client: Client, ids: List[str], delta: Dict[str, Any]):
+    """
+    Close (Dismiss) an alert in Prisma Cloud.
+
+    Args:
+        client: Demisto client.
+        ids: The IDs of the alerts to be dismissed.
+        delta: A dictionary of fields that changed from the last update - containing only the changed fields.
+
+    Returns: None.
+
+    """
+    close_notes = delta.get('closeNotes')
+    close_reason = delta.get('closeReason')
+    dismissal_note = f'{INCIDENT_OUTGOING_MIRROR_DISMISSAL_NOTE} - Closing Reason: {close_reason}, ' \
+                     f'Closing Notes: {close_notes}.'
+    time_filter = handle_time_filter(base_case=TIME_FILTER_BASE_CASE)
+
+    client.alert_dismiss_request(dismissal_note=dismissal_note, time_range=time_filter, alert_ids=ids)
+
+
+def reopen_alert_in_prisma_cloud(client: Client, ids: List[str]):
+    """
+    Re-open an alert in Prisma Cloud.
+
+    Args:
+        client: Demisto client.
+        ids: The IDs of the alerts to be re-opened.
+
+    Returns: None.
+
+    """
+    time_filter = handle_time_filter(base_case=TIME_FILTER_BASE_CASE)
+    client.alert_reopen_request(time_range=time_filter, alert_ids=ids)
+
+
+def whether_to_close_in_prisma_cloud(delta: Dict[str, Any]) -> bool:
+    """
+    Closing in the remote system should happen only when both:
+        1. The user asked for it.
+        2. One of the closing fields appears in the delta.
+
+    The second condition is mandatory so we will not send a closing request at all of the mirroring requests that happen
+    after closing an incident (in case where the incident is updated so there is a delta, but it is not the status
+    that was changed).
+
+    Args:
+        delta: A dictionary of fields that changed from the last update - containing only the changed fields.
+
+    Returns: True - if closing the prisma alert is needed, False - otherwise.
+
+    """
+    closing_fields = {'closeReason', 'closingUserId', 'closeNotes'}
+    return demisto.params().get('close_ticket') and any(field in delta for field in closing_fields)
+
+
+def whether_to_reopen_in_prisma_cloud(delta: Dict[str, Any]) -> bool:
+    """
+    Re-opening in the remote system should happen only when both:
+        1. The user asked for it.
+        2. The delta contains the 'closingUserId' field.
+
+    The second condition is mandatory so we will not send a re-opening request in case where the incident is updated so there is
+    a delta, but it is not the status that was changed.
+
+    Args:
+        delta: A dictionary of fields that changed from the last update - containing only the changed fields.
+
+    Returns: True - if re-opening the prisma alert is needed, False - otherwise.
+
+    """
+    return demisto.params().get('close_ticket') and 'closingUserId' in delta
```

Comment:
I would prefer to pass the specific part and not call it in internal command.
this is relevant to the other similar functions you introduced in the pr.